apps/web/src/screens/Dashboard/Embed.tsx

```diff
@@ -18,13 +19,28 @@ export const Embed: FC = () => {
 		return <TRexRunner />;
 	} else if (id === 'tezos') {
 		return <TezosDashboard />;
+	} else if (id === '000003') {
+		return (
+			<Image
+				style={styles.image}
+				source={{ uri: '/img/explore/ur-mock-layout.png' }}
+				resizeMode="cover"
+			/>
+		);
 	}
 
 	return (
-		<Stack>
+		<View>
 			<Text>EmbeddedApp</Text>
-		</Stack>
+		</View>
 	);
 };
 
 export default Embed;
+
+const styles = StyleSheet.create({
+	image: {
+		width: 352,
+		height: 600,
+	},
+});
```

Comment:
can you make it full screen

Reply:
this is a mock image of Under Realm layout which contain in Dashboard view, therefore it can not be full-screen